.gitignore

```diff
@@ -5,3 +5,8 @@ cache
 cscope*
 tags
 *.swp
+protos/*.cc
+protos/*.h
+*.jar
+*.iml
+.ijwb/
```

Comment:
What file would end with `.ijwb`?

Reply:
Latest IntelliJ IDEA uses this folder to place its configuration files.